lib/utils/tsv.rb

```diff
@@ -79,25 +87,65 @@ def self.print_category_row(file_handle, gene)
     end
 
     def self.print_interaction_header(file_handle)
-      header = ['entrez_gene_symbol','gene_long_name','interaction_claim_source',
-        'interaction_types','drug_name','drug_primary_name'].join("\t")
+      header = ['gene_name','gene_claim_name','entrez_id','interaction_claim_source',
+        'interaction_types','drug_claim_name','drug_claim_primary_name','drug_name','drug_chembl_id'].join("\t")
       file_handle.puts(header)
     end
 
-    def self.print_interaction_row(file_handle, gene)
-      gene.gene_claims.flat_map(&:interaction_claims).each do |interaction|
-        next if license_restricted? interaction.source.source_db_name
-        row = [
-           gene.name,
-           gene.long_name,
-           interaction.source.source_db_name,
-           interaction.interaction_claim_types.map(&:type).join(','),
-           interaction.drug_claim.name,
-           interaction.drug_claim.primary_name
-        ].join("\t")
+    def self.print_interaction_row(file_handle, interaction_claim)
+      return if license_restricted? interaction_claim.source.source_db_name
+      return unless interaction_claim.interaction_id
```

Comment:
We will want all interaction claims, not just grouped ones.

Reply:
This has been fixed.